rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -454,9 +465,9 @@ private int retryGetEndStatus(URI endPath, boolean compensate) {
                 // since this method is called from the recovery thread do not block
                 AsyncInvoker asyncInvoker = target.request()
                         .header(LRA_HTTP_CONTEXT_HEADER, lraId.toASCIIString())
+                        .header(LRA_HTTP_PARENT_CONTEXT_HEADER, parentId)
                         .property(LRA_HTTP_PARENT_CONTEXT_HEADER, parentId) // make the context available to participants
                         .header(LRA_HTTP_RECOVERY_HEADER, recoveryURI.toASCIIString())
-                        .property(LRA_HTTP_CONTEXT_HEADER, lraId)  // make the context available to the jaxrs filters
```

Comment:
this is LRA id, isn't this something that filters really rely on?

Reply:
+1